src/Evolution/Systems/CurvedScalarWave/Worldtube/ElementActions/InitializeCurrentIteration.hpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Protocols/Mutator.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+
+namespace CurvedScalarWave::Worldtube::Initialization {
+
+/*!
+ * \brief Sets the initial value of `CurrentIteration to 0.
+ */
+struct InitializeCurrentIteration : tt::ConformsTo<db::protocols::Mutator> {
```

Comment:
Did you want to add this to the executable?

Reply:
yes, it is added in #5855